### DISCUSSION TITLE
Objects are scaled while sliced

### DISCUSSION THREAD

**Discussion by kmeldner** [OrcaSlicer/OrcaSlicer#4874] [Dev]
Hi,
I have a strange situation.
When I slice, object are getting scaled (approx. 10x in X and Y).
As a consequence I am not able to print anything, as toolpaths are not as per model geometry.

I have tried restarting the computer. no improvement. problem is still there.

Any thoughts?

Cris.

**Comment by vgdh** [comment]
can you attach your project file?

**Comment by kmeldner** [comment]
I was doing pressure advance calibration and model was created by Orca slicer.

![image](https://github.com/SoftFever/OrcaSlicer/assets/55151881/11442788-435a-48bb-bced-42d210995afa)

Cris.
[Pressure Advance Test.zip](https://github.com/SoftFever/OrcaSlicer/files/14881727/Pressure.Advance.Test.zip)

**Comment by kmeldner** [comment]
Update,

When I turned computer this morning and sliced model in Orca it worked fine.

But than I tried to slice model from pressure advance calibration and it was scaled in X and Y during slice.
Than I tried with another model of mine and it was also sliced scaled.

So it seems that this problem is triggered by entering pressure advance calibration and slicing model generated for this test (PA tower).

Cris.

**Reply by vgdh** [reply]
do you use the latest 2.0 version?

**Comment by kmeldner** [comment]
yes

**Comment by vgdh** [comment]
yeah, there is a problem
can you create an ISSUE for it?

**Reply by kmeldner** [reply]
Ho do I do that?

**Reply by foreachthing** [reply]
https://github.com/SoftFever/OrcaSlicer/issues/new/choose
